### PR TITLE
fix: campaign launch

### PR DESCRIPTION
**Pull request by kupermind**
- Campaign launch.

### CODE REVIEW

File: contracts/MemeBase.sol [hunk 1]
```diff
@@ -77,9 +77,6 @@ contract MemeBase is MemeFactory {
         uint256 memeAmountForLP = (totalSupply * LP_PERCENTAGE) / 100;
         uint256 heartersAmount = totalSupply - memeAmountForLP;
 
-        // Wrap native token to its ERC-20 version, where applicable
```
Comment (kupermind) [inline contracts/MemeBase.sol:80]: Already wrapped previously

File: package.json [hunk 2]
```diff
@@ -44,7 +44,6 @@
     "@ethersproject/solidity": "^5.7.0",
     "@ethersproject/wallet": "^5.7.0",
     "eth-permit": "^0.2.3",
-    "solhint": "^3.6.2",
-    "@openzeppelin/contracts": "4.6.0"
```
Comment (kupermind) [inline package.json:48]: Not needed, a leftover

File: test/MemeBaseLaunchCampaign.js [hunk 3]
```diff
@@ -75,7 +75,6 @@ const main = async () => {
 
     // Unleash the meme token
     await memeBase.unleashThisMeme(memeToken);
-    return;
```
Comment (kupermind) [inline test/MemeBaseLaunchCampaign.js:78]: Continue the full test

File: contracts/MemeBase.sol (hunk 1 above)
Comment (DavidMinarsch) [reply contracts/MemeBase.sol:80]: correct

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**